dumper-integration-tests/redshift-tests/src/main/java/com/google/edwmigration/dumper/base/TestConstants.java

```diff
@@ -14,24 +13,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.google.base;
+package com.google.edwmigration.dumper.base;
 
 import static java.lang.System.getenv;
 
 import java.util.regex.Pattern;
 
-/**
- * Stores constants common among all tests.
- */
+/** Stores constants common among all tests. */
 public final class TestConstants {
 
   public static final String URL_DB = getenv("DB_URL");
   public static final String USERNAME_DB = getenv("USERNAME");
   public static final String PASSWORD_DB = getenv("PASSWORD");
 
-  public static final String ET_OUTPUT_PATH = getenv("EXPORT_PATH");
+  public static final String EXPORTED_FILES_BASE_PATH = getenv("EXPORT_PATH");
```

Comment:
Is this path guaranteed to be an `inputs.dir` at Gradle level? That would be required for this test to be valid. What execution stage causes the files to be placed in this path, and what is its proposed value?

Reply:
We will define these env vars in `kokoro_build.sh`.
That's going to be the integration point between Kokoro and the Test Framework.

This is how we did it for Teradata in the `kokoro_build.sh` file:
```
export EXPORT_PATH="${KOKORO_ARTIFACTS_DIR}/github/dwh-assessment-extraction-tool/bazel-bin/dist/dwh-assessment-extraction-tool/output/"
```

We can put some notes about it in README.md

Reply:
Looks like this is addressed in latest revision.